static/src/js/util/getS3Client.js

```diff
@@ -0,0 +1,19 @@
+import { S3Client } from '@aws-sdk/client-s3'
+
+/**
+ * Returns an S3 Client instance
+ */
+export const getS3Client = () => {
+  const config = {}
+
+  if (process.env.IS_OFFLINE) {
+    config.endpoint = 'http://localhost:4569'
+    config.forcePathStyle = true
```

Comment:
On MMT we set this value for both offline and deployed versions, why is it only for offline mode here?

Reply:
This was one of the things that was pointed out by NGAP. Having `forcePathStyle = true` changes the way the AWS SDK generates S3 URL: `https://s3.<region>.amazonaws.com/<bucket-name>/<key>`.  But in the deployed app, CloudFront expects the S3 origin to use virtual-hosted-style URLs `https://<bucket-name>.s3.<region>.amazonaws.com/<key>`.

According to AWS doc using forcePathStyle creates a conflict with CloudFront because it routes traffic based on the hostname, which must include the bucket name. Path-style URLs are incompatible with CloudFront's expected configuration.

For local where CloudFront isn't in use, forcePathStyle is needed to work to local S3.